Makefile

```diff
@@ -8,16 +8,25 @@ ns ?= "test"
 
 # Determine the operating system
 UNAME_S := $(shell uname -s)
+PHP_API_VERSION := $(shell php -i | grep 'PHP API' | awk -F' => ' '{print $$2}')
```

Comment:
This line is not working for me on Ubuntu 22.04. This regexp should be a more reliable substitute:
`php -i | grep -Po '(?<=PHP API => ).*'`